src/Exceptions/InvalidSize.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Spatie\PdfToImage\Exceptions;
+
+class InvalidSize extends \Exception
```

Comment:
Nitpick: let's import `Exception` so we can lose that `\`

Reply:
Nice catch! :+1: